azuredeploy.json

```diff
@@ -97,7 +104,8 @@
     "databaseSkuTier": "GeneralPurpose",
     "databaseSkuFamily": "Gen5",
     "databaseSkuName": "[concat('GP_', variables('databaseSkuFamily'), '_', parameters('databaseCores'))]",
-    "databaseConnectionString": "[concat('pgsql://', parameters('adminUserName'), '%40', variables('databaseServerName'), ':', parameters('adminPassword'), '@', variables('databaseServerName'), '.postgres.database.azure.com:5432/', parameters('databaseName'))]",
+    "databaseUserCredentials" : "[concat(uriComponent(concat(parameters('adminUserName'), '@', variables('databaseServerName'))), ':', parameters('adminPassword'))]",
+    "databaseConnectionString": "[concat('pgsql://', databaseUserCredentials, '@', variables('databaseServerName'), '.postgres.database.azure.com:', parameters('databaseServerPort'), '/', parameters('databaseName'))]",
```

Comment:
Nice idea to pull out parts of the connection string! I wonder if we should also pull out `variables('databaseServerName'), '.postgres.database.azure.com` into something like `databaseFqdn` for further improved readability.

Reply:
done.